src/base/orgs/TransferOwnership.svelte

```diff
@@ -38,10 +38,20 @@
   const onSubmit = async () => {
     assert(newOwner);
 
+    // Check if newOwner is valid ENS label or Ethereum address
     if (! utils.isAddress(newOwner)) {
-      state = State.Failed;
-      error = `"${newOwner}" is not a valid Ethereum address.`;
-      return;
+      if (! utils.isENSName(newOwner, config)) {
+        newOwner = `${newOwner}.${config.registrar.domain}`;
```

Comment:
This behavior is strange: if it's not an ENS name, it adds "radicle.eth" to the input field.

I would instead go to a failed state here, if we have neither an address nor an ENS name.

Reply:
EDIT: See comments below, I think it's best we just remove this behavior for now.